src/about.cpp

```diff
@@ -10,8 +11,19 @@ About::About(QWidget *parent) :
 {
     ui->setupUi(this);
     auto htmlText = ui->aboutText->toHtml();
```

Comment:
Ho!!!

This make me realize we already have a system to load different resources depending on the lang.
The QTResource system can load different resources depending of the lang (https://doc.qt.io/qt-5/resources.html)
This is already what we do for this file. The about text is translated to fr here (https://github.com/kiwix/kiwix-desktop/tree/master/resources/texts/fr) and depending of the lang, qt will load the english version or the french version if we are in french locale (because of this https://github.com/kiwix/kiwix-desktop/blob/master/resources/kiwix.qrc#L76-L78).

We could use the same system to load the translation in the json file.
And keep this whole htmlText in one file instead of integrate it in the json file.

Reply:
> We could use the same system to load the translation in the json file.
> And keep this whole htmlText in one file instead of integrate it in the json file.

Not sure to understand

Reply:
Qt already has a system to load a resource file specific to a lang. This is already use for the `about.html` resource. We have a "default" one in english (in resources/texts/about.html) and one specific to french in resources/texts/fr/about.html.
We could have the same things for the json file containing the translation. We could have a json file (texts.json) in resources/i18n and a texts.json per lang (resources/i18n/<lang>/texts.json). This way we simply would have to ask to qt to load the resource and it will load the right resource for us.

But saying that, it wouldn't protect us from a not up to date translation (with missing key).
Let's move with this PR. We have to think a bit more to see how (and if) we integrate the resource system.

Reply:
OK I see, I'll open a ticket about that